### PR TITLE
feature/files and directories

### PR DESCRIPTION
**Pull request by SidnySilva**

### CODE REVIEW

**Review by penguinuux — APPROVED** [review]

File: app/routes/__init__.py [hunk 1]
```diff
@@ -1,8 +1,16 @@
-from flask import Blueprint, Flask
+from flask import Flask
 
-bp_api = Blueprint("bp_api", __name__)
```
Comment (marcelosj3) [inline app/routes/__init__.py:3]: I believe this approach is more suitable if amidst the project we want to add a prefix to the whole api, such as `/api/users`.

File: app/routes/__init__.py [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+def init_app(app: Flask):
```
Comment (marcelosj3) [inline app/routes/__init__.py:5]: Using the above approach we could register all blueprints into the bp_api and then register the bp_api to the app, but if yous decided this way would be better, just let me know and i resolve these comments.

**Review by marcelosj3 — CHANGES_REQUESTED** [review]
Also, i think the exceptions could be its own module, so we could create generic ones for most of the requests and specific ones for each route that we feel like.

And don't forget to use `isort .` and then `black .` to maintain a code pattern.

I've just realised that one of your commits is missing the convetional commit prefix, might wanna rebase and rename it.

File: app/routes/__init__.py (hunk 1 above)
Comment (SidnySilva) [reply app/routes/__init__.py:3]: Done.

**Review by penguinuux — APPROVED** [review]

**Review by marcelosj3 — APPROVED** [review]

**Pull request merged**